apps/mitt-konto/src/Tracking.purs

```diff
@@ -1,13 +1,64 @@
 module Tracking where
 
+import Data.DateTime (DateTime)
+import Data.Maybe (Maybe(..))
 import Effect (Effect)
 import Foreign (Foreign)
 import Prelude (Unit)
+import Effect.Uncurried (EffectFn2, EffectFn3, EffectFn5, runEffectFn2, runEffectFn3, runEffectFn5)
+import KSF.Helpers as Helpers
 
-foreign import data Tracker :: Type
-foreign import newTracker :: Effect Tracker
-foreign import pushPageLoad :: Tracker -> Effect Unit
+foreign import login_ :: EffectFn3 Cusno LoginMethod Result Unit
+foreign import reclamation_ :: EffectFn5 Cusno Subsno DateString Claim Result Unit
+foreign import tempAddressChange_ :: EffectFn5 Cusno Subsno StartDateString EndDateString Result Unit
+foreign import pauseSubscription_ :: EffectFn5 Cusno Subsno StartDateString EndDateString Result Unit
+foreign import unpauseSubscription_ :: EffectFn3 Cusno Subsno Result Unit
+foreign import deleteTempAddressChange_ :: EffectFn5 Cusno Subsno StartDateString EndDateString Result Unit
+foreign import changeName_ :: EffectFn2 Cusno Result Unit
+foreign import changeAddress_ :: EffectFn2 Cusno Result Unit
+
+type Cusno = String
+type Subsno = String
+type DateString = String
+type Claim = String
+type StartDate = DateTime
+type StartDateString = String
+type EndDate = DateTime
+type EndDateString = String
+type Result = String
+type LoginMethod = String
+
+reclamation :: Cusno -> Subsno -> DateTime -> Claim -> Result -> Effect Unit
+reclamation cusno subsno date claim result =
+  runEffectFn5 reclamation_ cusno subsno (Helpers.formatDate date) claim result
+
+tempAddressChange :: Cusno -> Subsno -> StartDate -> Maybe EndDate -> Result -> Effect Unit
+tempAddressChange cusno subsno startDate endDate result = case endDate of
+  Just endDate' -> runEffectFn5 tempAddressChange_ cusno subsno (Helpers.formatDate startDate) (Helpers.formatDate endDate') result
```

Comment:
This is a bit verbose. It could be reduced to
```
let endDateString = maybe "indefinite" Helpers.formatDate endDate
in runEffectFn5 tempAddressChange_ cusno subsno (Helpers.formatDate startDate) endDateString result
```